README.md

```diff
@@ -1 +1,134 @@
-# Javascript-Group-Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse JavaScript Capstone Project</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [JavaScript-Group-Capstone] <a name="about-project"></a>
+
+
+**[JavaScript-Group-Capstone]** is a basic website that displays a number of movies. A user can like, or comment on a movie they choose. We will achieve that by using JavaScript to dynamically modify the DOM.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Html
+- css
+- Javascript
+- Visual Studio Code
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Home_Page]**
+- **[Comments_Popup]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://victoria129.github.io/Javascript-Group-Capstone/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+1. clone the project from link: https://github.com/Victoria129/Javascript-Group-Capstone.git by using git clone (link).
+2. open it on ur vs code and start using it. 
+
+
+### Usage
+
+To run the project, execute the following command:
+
+npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+npm test
+
+### Deployment
+
+You can deploy this project using:
+
+npm i -D gh-pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@girma336](https://github.com/girma336)
+
+👤 **Author2**
```

Comment:
- [ ] OPTIONAL
Generally, the subheading text _Author1_ and _Author2_ should be replaced with your respective full names in the ReadMe.